### DISCUSSION TITLE
Double click on Door to open a new map

### DISCUSSION THREAD

**Discussion by Daza99** [luiscastilho/dungeoneering#14] [Ideas]
Once we create a door it would be neat if we could double LMB click on it to bring up a open map box and we can set the map which is likely an interior map or perhaps it is a portal to another exterior map. And on the new map we simply do the same to link back to the previous map if it is possible for the players to go back outside. In DM mode hovering the pointer over a door you have setup could bring up name of the map that is assigned to it, which both reminds the DM or Solo player which map it loads and also that there is one stored. 

An additional idea, in the popup window to choose a map, there could be a tick box we can click on instead [ ] No map but display this text instead.  And clicking that has a text box we can write something if we wish to. Which might be a note for the DM, it might be LOCKED needs the Skeleton Key or Cannot be opened etc.

**Comment by luiscastilho** [comment]
Great ideas! I never thought dungeoneering might be useful for solo TTRPG. Makes sense. :)

I will add these ideas to the TODO list. Thanks!
